src/main/java/seedu/dengue/model/person/Postal.java

```diff
@@ -21,9 +21,15 @@ public class Postal {
      * @param postal A valid postal number.
      */
     public Postal(String postal) {
+        boolean hasStart = postal.substring(0, 1).equals("S");
         requireNonNull(postal);
         checkArgument(isValidPostal(postal), MESSAGE_CONSTRAINTS);
-        value = postal;
+        if (hasStart) {
```

Comment:
This should be on a different level of abstraction. Also are we storing with the S or without? I was thinking stripping to the integer value and then we display as such:
`S(<number>)`

Reply:
seems right